config/rbac/topolvm_node_scc.yaml

```diff
@@ -0,0 +1,30 @@
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: topolvm-node
+  annotations:
+    kubernetes.io/description: 'privileged allows access to all privileged and host
+        features and the ability to run as any user, any group, any fsGroup, and with
+        any SELinux context.  WARNING: this is the most relaxed SCC and should be used
+        only for cluster administration. Grant with caution.'
+allowHostDirVolumePlugin: true
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPID: true
```

Comment:
Why?

Reply:
https://github.com/red-hat-storage/lvm-operator/blob/848792e3cda401d0567e0140d006b068e3c54adf/controllers/topolvm_node.go#L182

Reply:
That does not tell why we need `HostPID`.

Reply:
@leseb generally nodeplugin doesn't need it however lvmd in the daemonset needs this.

It uses `nsenter` to get into host pid nasemspace and run lvm ops

Reply:
I don't think you need `HostPID` for `nsenter` to run any commands on the host. As far as I can tell, Rook uses nsenter already to exec command on the host and we don't use `HostPID`.

However, if you really feel like we do then let's comment on why. Thanks

Reply:
Extract from [Openshift documentation](https://docs.openshift.com/container-platform/4.9/rest_api/security_apis/securitycontextconstraints-security-openshift-io-v1.html#specification):
```
allowHostPID | boolean | AllowHostPID determines if the policy allows host pid in the containers.
-- | -- | --
```
If you are asking why do we need hostpid: true in the pod, this is because lvmd needs to see the pid:1 on the host. ([here you can follow the complete discussion that drove to this point](https://github.com/topolvm/topolvm/issues/139#issuecomment-684069197))

But, for me is new this world of SCCs.. how do you suggest to add this requirement (see pid 1 in the host) in the SCC?

Reply:
I don't mean to rehearse or argue on the past decision that led to enabling `HostPID`. I just want to make sure everybody knows and understands why it is needed. That's all :) So in the `config/rbac/topolvm_node_scc.yaml` I'm just suggesting adding more comments to explain why each option we enable is needed.